bots/alpha_beta_minimax.py

```diff
@@ -0,0 +1,47 @@
+def minimax(board, depth, eval, alpha, beta, maximizing_player):
```

Comment:
If you want to, some unit tests or some integration tests for this method would be nice.
If you don't want to write tests that's okay too.

Reply:
Sure thing I will try to implement them with pytest.